freeipa/src/main/resources/freeipa-salt/salt/upgradeccm/restoressl.sls

```diff
@@ -0,0 +1,17 @@
+restore_last_ssh_conf:
+  module.run:
+    - name: file.restore_backup
+    - path: /etc/nginx/sites-enabled/ssl.conf
+    - backup_id: 0
+
+touch_ssl_conf_for_restart:
```

Comment:
this might should require the previous state, so if that one fails, this wont run

Reply:
I find no way to do this